zanata-war/src/main/java/org/zanata/security/FacesSecurityEvents.java

```diff
@@ -34,6 +40,8 @@
  * Override the {@link org.jboss.seam.security.FacesSecurityEvents} component to
  * alter default values.
  *
+ * TODO [CDI] need to find a CDI replacement.
```

Comment:
This class extends org.jboss.seam.security.FacesSecurityEvents. Need to do it in deltaspike or something else. 

----- Original Message -----

> From: "Sean Flanigan" <notifications@github.com>
> To: "zanata/zanata-server" <zanata-server@noreply.github.com>
> Cc: "Patrick Huang" <pahuang@redhat.com>
> Sent: Thursday, July 30, 2015 5:24:34 PM
> Subject: Re: [zanata-server] Replace Seam Security (#908)

> In zanata-war/src/main/java/org/zanata/security/FacesSecurityEvents.java :
> > @@ -34,6 +40,8 @@
> >   * Override the {@link org.jboss.seam.security.FacesSecurityEvents}
> >   component to
> >   * alter default values.
> >   *
> > + * TODO [CDI] need to find a CDI replacement.

> for what?

> —
> Reply to this email directly or view it on GitHub .